etc/devices/redfishpower-cray-ex-rabbit.dev

```diff
@@ -4,7 +4,8 @@
 #
 # - If necessary, set your system's username/password in the login
 #   section of each specification below.  By default, a manufacturer
-#   default is set below.
+#   default is set below.  Or it can be configured via redfishpower's
+#   --auth option.
```

Comment:
Could we completely drop the suggestion that people should edit the dev files?  It causes problems.  Command line is better!